### PR TITLE
examples: Azure Storage examples

### PR DESCRIPTION
**Pull request by enghitalo**
- [ ] Use OpenTofu
- [ ] CI Test using Azurite
- [ ] examples/cloud/azure/storage/main_test.v
- [ ] examples/cloud/azure/storage/list_files_from_azure.v
- [ ] examples/cloud/azure/storage/download_file_from_azure.v
- [x] examples/cloud/azure/storage/send_file_to_azure.v
- [x] Terraform

* Sorry for this "incomplete" PR, I'm going to do some traveling this week, and work on some other projects and I don't want to risk losing this work.

### CODE REVIEW

File: vlib/time/format.v
Comment (enghitalo) [inline vlib/time/format.v:1]: I think should be better open it in another PR.

Comment (Delta456) [reply vlib/time/format.v:1]: Agreed

**Comment by spytheman** [comment]
Please work in your own repository, there is no need to trigger the CI of the main one, until you are ready to make a PR.
